polaris-react/src/components/Frame/Frame.module.css

```diff
@@ -435,4 +431,37 @@
     top: var(--pg-top-bar-height);
     background-color: var(--p-color-bg);
   }
+  position: fixed;
+  width: 100%;
+  transition: width var(--p-motion-duration-250) var(--p-motion-ease);
+
+  @media (prefers-reduced-motion) {
+    transition: none;
+  }
+
+  @media (--p-breakpoints-md-up) {
+    /* stylelint-disable-next-line polaris/conventions/polaris/custom-property-allowed-list -- private token from component */
+    width: calc(100% - var(--pc-frame-offset, 0px));
+  }
+
+  .hasSidebar & {
+    transition: width var(--p-motion-duration-250) var(--p-motion-ease);
```

Comment:
I have an issue https://github.com/Shopify/polaris-backlog/issues/1606 to update these to transforms for performance